corehq/apps/hqcase/tasks.py

```diff
@@ -23,11 +21,10 @@ def explode_cases(user_id, domain, factor, task=None):
 
     old_to_new = dict()
     child_cases = list()
+    accessor = CaseAccessors(domain)
 
-    case_ids = get_case_ids_in_domain_by_owner(
-        domain, owner_id__in=user.get_owner_ids(), closed=False)
-    cases = (CommCareCase.wrap(doc)
-             for doc in iter_docs(CommCareCase.get_db(), case_ids))
+    case_ids = accessor.get_case_ids_by_owners(user.get_owner_ids())  # TODO closed
```

Comment:
https://github.com/dimagi/commcare-hq/pull/10928/commits/d3e7255cc541138272754768104038c6f5b3886d and https://github.com/dimagi/commcare-hq/pull/10928/commits/e1dfa9c8d43928b7c3fc4d7d69b37457ec1a7cbe